src/capital-project/capital-project.service.ts

```diff
@@ -19,6 +20,16 @@ export class CapitalProjectService {
     private readonly capitalProjectRepository: CapitalProjectRepository,
   ) {}
 
+  async findCapitalProjects({
+    limit = 20,
+    offset = 0,
+  }: FindCapitalProjectsQueryParams) {
+    return await this.capitalProjectRepository.findCapitalProjects({
+      limit,
+      offset,
+    });
```

Comment:
Before returning the result of the query, we need to place the results in an object with the metadata about order and pagination